### DISCUSSION TITLE
Feature tab broken with 20240914 - Solved

### DISCUSSION THREAD

**Discussion by hjfisch** [rern/rAudio#1770] [Fixed Issues]
![Screenshot 2024-09-14 093157](https://github.com/user-attachments/assets/31cf67dc-b0e2-4fce-9d8e-b00324676d0e)

**Comment by rern** [comment]
Please try: `rm -f /srv/http/data/system/volume*`

**Comment by hjfisch** [comment]
Did not help

**Reply by rern** [reply]
Then try: 
`curl -L https://github.com/rern/rAudio/raw/UPDATE/srv/http/bash/settings/features-data.sh -o /srv/http/bash/settings/features-data.sh`

**Comment by hjfisch** [comment]
That's it .
Working now .
Update to 202409141 is working too .

**Reply by klassikmann** [reply]
I had the problem that the feature tab was not responding at all, not even producing an error message. The UPDATE of `features-data.sh` solved my problem also.

**Comment by hjfisch** [comment]
Due to the fact that everything I need is working for me now , an Image would be nice .
